communicator/ssh/communicator.go

```diff
@@ -96,6 +96,21 @@ func (c *Communicator) Connect(o terraform.UIOutput) (err error) {
 			c.connInfo.KeyFile != "",
 			c.connInfo.Agent,
 		))
+
+		if c.connInfo.BastionHost != "" {
+			o.Output(fmt.Sprintf(
+				"Using configured bastion host..."+
+					"  Host: %s\n"+
+					"  User: %s\n"+
+					"  Password: %t\n"+
```

Comment:
Note the password and private key outputs here are just booleans. So you are correct! :grinning: